python/hsfs/core/external_feature_group_engine.py

```diff
@@ -25,6 +26,12 @@ def save(self, feature_group):
             external_dataset = engine.get_instance().register_external_temporary_table(
                 feature_group, "read_ondmd"
             )
+            if external_dataset is None:
```

Comment:
What if user passes the features (schema)? in that case it's not going into the above if clause.

Reply:
fixed compute stats if features are passed